scripts/spack/devtools_configs/toss_4_x86_64_ib/spack.yaml

```diff
@@ -0,0 +1,253 @@
+spack:
+  config:
+    install_tree:
+      root: $spack/..
+      projections:
+        all: '{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+    misc_cache: $spack/../misc_cache
+    test_stage: $spack/../test_stage
+    build_stage::
+    - $spack/../build_stage
+
+  view:
+    default:
+      root: ../view
+      select: [llvm, cppcheck, doxygen, py-ats, py-sphinx]
+      projections:
+        all: '{name}-{version}'
+
+  compilers::
+  - compiler:
+      environment: {}
+      extra_rpaths: []
+      flags: {}
+      modules:
+      - gcc/10.3.1
+      operating_system: rhel8
+      paths:
+        cc: /usr/tce/packages/gcc/gcc-10.3.1/bin/gcc
+        cxx: /usr/tce/packages/gcc/gcc-10.3.1/bin/g++
+        f77: /usr/tce/packages/gcc/gcc-10.3.1/bin/gfortran
+        fc: /usr/tce/packages/gcc/gcc-10.3.1/bin/gfortran
+      spec: gcc@10.3.1
+      target: x86_64
+
+  packages:
+    all:
+      # This defaults us to machine specific flags of ivybridge which allows
+      # us to run on broadwell as well
+      target: [ivybridge]
+      compiler: [gcc, intel, pgi, clang, xl, nag]
+      providers:
+        blas: [openblas]
+        lapack: [netlib-lapack]
+        mpi: [mvapich2]
+        gl: [opengl]
+        glu: [openglu]
+
+    openblas:
+      buildable: false
+      externals:
+      - prefix: /usr
+        spec: openblas@0.3.15
+    opengl:
+      buildable: false
+      externals:
+      - spec: opengl@1.7.0
+        prefix: /usr
+    openglu:
+      buildable: false
+      externals:
+      - spec: openglu@1.3.1
+        prefix: /usr
+
+    # Lock down which MPI we are using
+    mpi:
+      buildable: false
+    mvapich2:
+      buildable: false
+      externals:
+      - spec: mvapich2@2.3.6%clang@14.0.6 process_managers=slurm arch=linux-rhel8-ivybridge
+        prefix: /usr/tce/packages/mvapich2/mvapich2-2.3.6-gcc-10.3.1
```

Comment:
```suggestion
```

Reply:
if we make this change, should i remove %gcc in the below spec line?